src/compounds/product-table/src/generics.ts

```diff
@@ -76,6 +78,16 @@ const formatters: { [unit: string]: (value: number) => string } = {
     if (megabytes >= 1000) return `${Math.floor(megabytes / 1000)}GB`
 
     return `${megabytes || 0}MB`
+  },
+
+  'exchange-rate': (value, count = 4) => {
```

Comment:
What is `count`? Is that the number of decimal places?

Reply:
Yes, it is. @samhstn